src/api/mock-api.ts

```diff
@@ -0,0 +1,7 @@
+import MockAdapter from "axios-mock-adapter";
```

Comment:
As discussed, lets have it in the readme, instead of adding a package that we might not use.